### PR TITLE
Adding functional test

### LINKED ISSUE #4388: Multiple Tag Subscription Functional Tests
Write functional tests for the multiple tag controller action.

### PR DESCRIPTION
**Pull request by SidharthBansal**
Fixes #4388 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] PR is descriptively titled 📑
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: test/functional/subscription_controller_test.rb [hunk 1]
```diff
@@ -27,4 +27,14 @@ def setup
     assert_response :success
   end
 
+  test 'should subscribe to multiple tags' do
+    UserSession.create(users(:bob))
+    assert users(:bob).following(:awesome)
+    get :multiple_add, params: { type: 'tag', names: 'blog,kites,,balloon,awesome' }
+    assert_response :redirect
+    assert users(:bob).following(:blog)
```
Comment (SidharthBansal) [inline test/functional/subscription_controller_test.rb]: If a user checks any tag which is already subscribed then it must remain subscribed to the user.

Comment (SidharthBansal) [reply test/functional/subscription_controller_test.rb]: Here tag `awesome` status remains `following` for the user `bob`.

**Comment by SidharthBansal** [comment]
@jywarren pinging you here so that we can move forward with Subscription project also.

**Comment by jywarren** [comment]
Love it! Great work.

**Pull request merged**